src/components/Core.js

```diff
@@ -185,17 +207,18 @@ const Core = (props) => {
                 src={"mob.near/widget/ProfileImage"}
                 props={{
                   accountId: account.accountId,
-                  className: "d-inline-block m-2",
-                  imageClassName: "rounded-circle w-100 h-100",
-                  style: { width: "42px", height: "42px" },
+                  className: "d-inline-block",
```

Comment:
This too, the goal is for most of these styles to be configurable from the root

```css
.button {
   
}

core__auth: {
    .dropdown {
         .dropdown-item {
         
         }
    }
}
```

This auth layer (Core) is especially relevant. Think no-code -- different apps (like trylivepeer.near.page, etc) will be sharing this auth layer, but each will want to customize in their own way.

Reply:
Last thing -- when thinking about extracting styles, when different apps implement their auth layer, they'll probably provide this as a widget (such as trylivepeer.near/widget/core) or something like that. So we only want to pull out *global* styles, that will be shared by these different widget implementations.

Reply:
done